frontend/public/components/secrets/create-secret.tsx

```diff
@@ -999,6 +1036,7 @@ type BaseEditSecretProps_ = {
   saveButtonText?: string;
   explanation: string;
   onCancel?: () => null;
+  onSave?:(name: string) => void;
```

Comment:
If you're adding a base prop type, you need to ensure that it's called in all cases otherwise the api isn't clear under what circumstances it is to be called.